python-sdk/src/astro/sql/operators/transform.py

```diff
@@ -100,10 +101,52 @@ def my_sql_statement(table1: Table, table2: Table, execution_date) -> Table:
     )
 
 
+def transform_sql(
```

Comment:
@feluelle, you're introducing a sort of "feature" (to allow users to view the rendered SQL in the UI) - but we don't have any tests that check this.  This means that, probably, if someone rolled back the change, the CI wouldn't complain about this change, and we'd lose the feature. Do you have any idea for how we could test this?

Reply:
Do you think it is useful to add a test checking the `template_fields` and `template_ext`?

Reply:
I think it is worth it, @feluelle. From one perspective, it feels like we're copying the implementation and test, but at least we'd be testing the "feature" (assuming Airflow doesn't change their interface, which I think is a valid assumption for now)

Reply:
Alright, will add it then 👍

Reply:
Added in 1face6c.